dropwizard-parent-pom/pom.xml

```diff
@@ -252,11 +251,6 @@
       </dependency>
       <dependency>
         <groupId>io.dropwizard</groupId>
-        <artifactId>dropwizard-spdy</artifactId>
```

Comment:
spdy was removed.  I looked through the other dependencies as noted in the dropwizard-bom (where they notate the actual versions now).